Nighttime-Operations/Weather-Constraints/index.rst

```diff
@@ -0,0 +1,179 @@
+.. _Nighttime-Operations-Weather-Constraints:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+.. _`Gemini weather station`: https://gemini.edu/sciops/telescopes-and-sites/weather/cerro-pachon/envmon/
+.. _`Gemini cloud cameras`: https://www.gemini.edu/sciops/telescopes-and-sites/weather/cerro-pachon/cloud-cam/stills.html
+
+
+###################
+Weather Constraints
+###################
+
+.. This section should provide a brief, top-level description of the page.
+
+This page defines the different types of operational constraints due to weather.
+
+Operating conditions used to define the Rubin construction project are found in `LTS-54 <https://ls.st/lts-54>`__. 
+
+As commissioning gets underway, the operating conditions will be better constrained and different situations will apply to the Main telescope and Auxiliary Telescope.
+
+.. toctree::
+    :maxdepth: 2
+    :titlesonly:
+    :glob:
+
+..    *
+
+.. _weather-constraints-deciding-to-open:
+
+Deciding to open
+^^^^^^^^^^^^^^^^
+Before opening the dome to take on-sky data or for cooling, observers must take care to inspect the weather conditions.
+Humidity (especially the dew point difference), cloud cover, and wind speed are the three largest determining factors for safety of the telescope and equipment. 
+
+Observers should inspect the forecast for the night, and review the weather trends on the Rubin weather station (currently unavailable) and the `Gemini weather station`_.
+The weather server on Gemini updates with new information every 30 seconds, so wind speeds are gusts, rather than sustained averages.
+
+Go outside and look for incoming cloud fronts, and inspect the buildings and cars for condensation if humidity is above 70%.
+Be more cautious to open if there will not be someone in the control room to montior changing weather conditions, e.g. opening before going to dinner or to complete another task.
+
+.. warning::
+    Vent gates:
+        The vent gates on the first floor of the AuxTel dome are not rated for high winds. 
+        The vent gates MUST be closed if the wind speed (gusts and sustained) is above 8 m/s.
+
+
+
+.. note::
+    Extraction fan use:
+        The extraction fan on vent gate 3 can operate at 50-100% power for initial cooling. 
+        During on-sky operations, the extraction fan must be run at 20-25% power.
+        If winds are over 8 m/s, the vent gate must be closed and the extraction fan turned off.
+        If humidity is rising quickly, or >= 65%, the extraction fan should be turned off to avoid drawing in more moisture through the dome. 
```

Comment:
65% was more of an instinct - I wasn't implying that closure should happen, just to not exacerbate the situation, since 70% is when we should be paying attention to take the decision to close. This limit might buy more time, especially if it isn't windy out. Testing would just be running the fan still and reviewing RH inside the dome. Or putting a metal surface near the fan and seeing if water condenses on it.

Reply:
Ok, maybe just add a (TBR) beside the 65%.